clientv3/namespace/doc.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2017 The etcd Authors
```

Comment:
can we also add a link to this godoc in https://github.com/coreos/etcd/tree/master/clientv3#etcdclientv3? I think this should be useful for a lot of users.